offchain-modules/cli/src/commands/dapp/mod.rs

```diff
@@ -16,6 +17,7 @@ pub async fn dapp_handle(command: DappCommand) -> Result<()> {
         DappCommand::CKBIndexer(args) => ckb_indexer(args).await,
         DappCommand::CkbTxRelayer(args) => ckb_tx_relay(args).await,
         DappCommand::EthTxRelayer(args) => eth_tx_relay(args).await,
+        DappCommand::AccountMonitor(args) => account_monitor(args).await,
```

Comment:
We don't need another monitor process, better use existed `RelayerMonitor` process